packages/replay/test/integration/sampling.test.ts

```diff
@@ -20,14 +23,58 @@ describe('Integration | sampling', () => {
     const spyAddListeners = jest.spyOn(replay, '_addListeners');
     jest.runAllTimers();
 
-    expect(replay.session?.sampled).toBe(false);
-    expect(replay.getContext()).toEqual(
-      expect.objectContaining({
-        initialTimestamp: expect.any(Number),
-        initialUrl: 'http://localhost/',
-      }),
-    );
+    expect(replay.session).toBe(undefined);
+
+    // This is what the `_context` member is initialized with
+    expect(replay.getContext()).toEqual({
+      errorIds: new Set(),
+      traceIds: new Set(),
+      urls: [],
+      earliestEvent: null,
+      initialTimestamp: expect.any(Number),
+      initialUrl: '',
+    });
     expect(mockRecord).not.toHaveBeenCalled();
     expect(spyAddListeners).not.toHaveBeenCalled();
+
+    // TODO: Should we initialize recordingMode to something else?
```

Comment:
not sure I understand this TODO comment, you mean for the test? Why?

Reply:
Ah I think I was wondering if we should initialize recordingMode to `null` or something since in this test case sample rates are both 0, so plugin is in a disabled state but recordingMode is `session`.